databricks/koalas/ml.py

```diff
@@ -31,10 +42,15 @@ def corr(kdf, method='pearson'):
                    * pearson : standard correlation coefficient
                    * spearman : Spearman rank correlation
     :return: :class:`pandas.DataFrame`
+
+    >>> ks.DataFrame({'A': [0, 1], 'B': [1, 0], 'C': ['x', 'y']}).corr()
+         A    B
+    A  1.0 -1.0
+    B -1.0  1.0
     """
-    assert method in ('pearson', 'spearman'), method
+    assert method in ('pearson', 'spearman')
```

Comment:
why?

Reply:
What was the point of the `, method` at the end of the line?